bucket/bin/activate

```diff
@@ -0,0 +1,78 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
You are not supposed to push your `virtual environment` folder. It's more appropriate you make use of `VirtualEnvWrapper` to avoid things like this.